src/main/resources/static/css/home-page-style.css

```diff
@@ -0,0 +1,77 @@
+body {
+    background-color: #42BFDD;
+}
+
+p {
+    font-size: 24px;
+}
+
+h2 {
+    font-size: 48px;
+}
+
+.button {
+    background-color: #1D77C3;
+    border: none;
+    color: white;
+    padding: 15px 50px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    width: 85%;
+    gap: min(2px, 1%);
+}
+
+.button:hover {
+    cursor: pointer;
+    background-color: lightslategray;
+  }
+
+.grid-container {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    grid-column-gap: 10px;
+    grid-row-gap: 10px;
+    height: 100vh;
+    width: 100%;
+}
+
+.grid-item-2-img {
+    width: 100%;
+    height: auto;
+}
+
+.grid-item-4-img {
+    width: 100%;
+    height: auto;
+}
+
+.grid-item-1 {
+    grid-column: 1 / 4;
+    grid-row: 1 / 2;
+    background-color: #D8E9F7;
+    text-align: center;
+}
+
+.grid-item-2 {
+    grid-column: 1 / 2;
+    grid-row: 2 / 4;
+    background-color: #D8E9F7;
+    text-align: center;
+}
+
+.grid-item-3 {
+    grid-column: 2 / 4;
+    grid-row: 2 / 3;
+    background-color: #D8E9F7;
+    text-align: center;
+}
+
+.grid-item-4 {
+    grid-column: 2 / 4;
+    grid-row: 3 / 4;
+    background-color: #D8E9F7;
+    text-align: center;
+}
```

Comment:
Could we have a newline after this - text files should always end with a newline